pegasus/sites.v3/code.org/views/congrats_share.haml

```diff
@@ -1,9 +1,31 @@
+:javascript
+  // Triggers the registration of a Google Analytics "share_hoc_certificate" event.
+  // Args:
+  //   social_platform: The social platform the HOC certificate is being shared on. 
+  function trackSocialEvent(social_platform) {
+    // Determine the tutorial being shared via the document referrer.
+    if (!document.referrer) {
```

Comment:
We might want to think about a more robust/maintainable way of doing this longer term-- we shouldn't have to parse it out of the referer. This is fine for now however.